daemon/daemon_unix.go

```diff
@@ -455,12 +455,19 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 		ipamV4Conf.AuxAddresses["DefaultGatewayIPv4"] = config.Bridge.DefaultGatewayIPv4.String()
 	}
 
-	var ipamV6Conf *libnetwork.IpamConf
+	var (
+		ipamV6Conf     *libnetwork.IpamConf
+		deferIPv6Alloc bool
+	)
 	if config.Bridge.FixedCIDRv6 != "" {
 		_, fCIDRv6, err := net.ParseCIDR(config.Bridge.FixedCIDRv6)
 		if err != nil {
 			return err
 		}
+
+		ones, _ := fCIDRv6.Mask.Size()
+		deferIPv6Alloc = ones <= 80
```

Comment:
@icecrime 
When IPv6 support was added into docker, the IPv6 address assignment could happen in two ways:
- from MAC address (last 48 bits of the IPv6 address == MAC address)
- or via ip allocator

The first method was the preferred way, always chosen by bridge driver when the IPv6 network had enough host bits to contain the MAC address. That's from where the 80 (128-48) bits comes from.

This logic gave a backdoor access to configure deterministic IPv6 addresses for containers.
In order to restore that behavior (which got broken) only for the default bridge network, I had to bring the 80 bits in here.
Whenever we support the IPAM options for endpoint (container), we can deprecate the above behavior.

Since the `ones` are not visible outside of the if scope, I opted for a bool variable outside.
If I had to initialize the NetworkOption setter function inside the if scope, then I'd need to define anyway an extra variable outside, of type `libnetwork.NetworkOption`. Hence I did not see a difference.